src/Components/AlertBadge/AlertBadge.tsx

```diff
@@ -29,12 +29,14 @@ const AlertBadge: React.FC<IAlertBadgeProps> = ({
                     ? classNames.groupContainer
                     : classNames.singleContainer
             }
-            onMouseOver={(element) => {
+            onMouseMove={(element) => {
```

Comment:
won't onMouseMove fire super frequently? Do we really want it to keep firing for every movement even when we are just hovering the same badge?

Reply:
can we use `onMouseEnter` instead? We have had better luck with that in other places since it only fires a single time on entry.

Reply:
No, mouseMove is fine because the popover is over the top of it.

Reply:
We could possibly revert this change as I'm pretty sure its the true/false flag that was causing the issue

Reply:
reverted